src/api/app/controllers/concerns/triggerable.rb

```diff
@@ -11,13 +11,16 @@ def set_project
     raise Project::Errors::UnknownObjectError, "Sorry, triggering tokens for remote project \"#{@project_name}\" is not possible." unless @project.is_a?(Project)
   end
 
-  def set_package
+  # rubocop:disable Naming/AccessorMethodName
+  def set_package(package_find_options: {})
+    # rubocop:enable Naming/AccessorMethodName
```

Comment:
```suggestion
  def set_package(package_find_options: {}) # rubocop:disable Naming/AccessorMethodName
```

I think this works, too...

Reply:
done @danidoni